tools/makeusbconf.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# Usage:
+#
+#      ./makeusbconf.sh [-d] [-i] [-s <size in Kb>] [-f <file> ] <output.img>
+#
+MKFLASH_TAG="$(linuxkit pkg show-tag pkg/mkimage-raw-efi)"
+
+usage() {
+    echo "Usage: $0 [-d] [-i] [-s <size in Kb>] [-f <file> ] <output.img>"
+}
+
+cleanup() {
+    rm $TMPDIR/* 2>/dev/null
+    rmdir $TMPDIR
+}
+
+SIZE=204800
+TMPDIR=$(mktemp -d)
+
+while getopts dif:s: o
+do      case "$o" in
+        d)      mkdir $TMPDIR/dump;;
+        i)      mkdir $TMPDIR/identity;;
+        f)      cp $OPTARG $TMPDIR/usb.json;;
```

Comment:
Why not check -f a file named OPTARG exists? Not user friendly to have to read the script source code to figure that out.